napari/utils/colormaps/colormap.py

```diff
@@ -1,7 +1,7 @@
 from enum import Enum
 
 import numpy as np
-from pydantic import validator
+from pydantic import root_validator, validator
 
 from ..events import EventedModel
 from ..events.custom_types import Array
```

Comment:
Can you add to the doc string to explain the new behaviour where we add coordinates